foqus_lib/framework/graph/graph.py

```diff
@@ -401,9 +436,13 @@ def generateGlobalVariables(self):
         """
         self.x = self.input.createOldStyleDict()
         self.f = self.output.createOldStyleDict()
+        self.xvector = self.input_vectorlist.createOldStyleDict()
+        self.fvector = self.output_vectorlist.createOldStyleDict()
         # x and f are ordered dictionaries so keys are already sorted
         self.xnames = list(self.x.keys())  # get a list of input names
         self.fnames = list(self.f.keys())  # get a list of output names
+        # self.xvectornames = list(self.xvector.keys())  # get a list of input vector names
+        # self.fvectornames = list(self.fvector.keys())  # get a list of output vector names
```

Comment:
Go ahead and delete commented code.

Reply:
Yes I will delete this. Thank you @eslickj